File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoArgValueContextPluginConfig.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdComposite;
import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoArgValueContextPluginConfig extends StudioDtoArgValueContext
{
  @Nullable
  public MetaIdComposite compositeId;

  public MetaIdField fieldId;

  @Nullable
  public String[] valuePathArray;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoArgValueContextPluginConfig.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdComposite
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.dto.StudioDtoArgValueContext

interface StudioDtoArgValueContextPluginConfig : StudioDtoArgValueContext
{
  val compositeId: MetaIdComposite?
  val fieldId: MetaIdField
  val valuePathArray: Array<String>?
}